howtos/versioning.md

```diff
@@ -1,14 +1,14 @@
 # Versioning
 
-Our documentation is versioned. All sections are versioned together, this includes products that do not have a fixed release cycle (such as Cloud Console).
+Our documentation is versioned. All sections are versioned together, this includes products that do not have a fixed release cycle (such as Console, Web Modeler, and Connectors).
```

Comment:
Similar to my last comment -- there is more to say here re: Optimize. Would you like it here as feedback on this PR, or may I tackle it later as part of #1116?

Reply:
Tackle it in #1116.